django/db/models/fields/files.py

```diff
@@ -75,6 +75,7 @@ def open(self, mode='rb'):
             self.file.open(mode)
         else:
             self.file = self.storage.open(self.name, mode)
+        return self
```

Comment:
Yes, it would be tested indirectly via `FileField`, probably. A test might go in `tests/model_fields/test_filefield.py`.